src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -289,6 +290,36 @@ public class EntityFrameworkServices : IEntityFrameworkService
             return true;
         }
 
+        // Look for the model type in the current project. 
+        // If its not found in the current project, look in the dependencies.
+        private Type GetModelTypeFromAssembly(string modelTypeName, Assembly assembly)
+        {
+            Type modelType = assembly.GetType(modelTypeName);
+            if (modelType == null)
+            {
+                // Need to look in the dependencies of this project now.
+                var dependencies = assembly.GetReferencedAssemblies().GetEnumerator();
```

Comment:
I don't remember , but does build happens before we do that? If not assume user added a new model in the project B and did not build yet, then tries to scaffold in project A, which would find Project B assembly, but no new type yet... What is the plan for this case?